### ISSUE TITLE
Optimise SEO to make site visible in search engnines

### ISSUE THREAD

**Issue by MasssiveJuice08** [morrowind-modding/morrowind-modding.github.io#27]
The wiki is currently not visible in Google search results.

One possible solution to help with this is consistent naming of folder and `.md` files per the screenshot attached:

![Discord-Quartz_URL-Slug_2023-11-15.jpg](https://github.com/morrowind-modding/morrowind-modding.github.io/assets/122503558/1c82603e-12f5-467a-9a8a-7f6fa8db961f)

> "lowercase with dashes, only using english alphabet (26 letters) and numbers: `folder-for-inbox`; `file-with-poems.md`"

Markdown files would then need `title` and `aliases` frontmatter added to render the titles in the wiki site without dashes `-` for spaces, e.g., so it displays an article title as `File with Poems` instead of `file-with-poems`

**Comment by magicaldave** [comment]
We could probably improve SEO for sure, but I think there may be something deeper going on here with Google specifically. We're actually the first result in Bing, Yahoo, DuckDuckGo, and Ecosia. No trace of the site on Google though.

**Comment by MasssiveJuice08** [comment]
Oh that's wild. I didn't even think to check other search engines 😂

I agree there's more going on here with Google. The principle of 'properly naming' is still useful though. For example, giving image files a descriptive file name like `teacup_on_a_table.png` instead of `gkdosfjdjsdidofi.png` helps search engine indexing (as well as accessibilty for screen readers).

[Naming images and alt text for SEO](https://morrowind-modding-with-massivejuice.vercel.app/01-fleeting-notes/naming-images-and-alt-text-for-seo/)

**Comment by MasssiveJuice08** [comment]
Mind you, I did shoot us in the foot accessibility-wise by integrating SIRvb's Image Adjustments, as that utilises image 'alt text' as keys to apply CSS classes to images to style them. If a screenreader read the alt text of our images it would read "wmed right profile p+b", which is a series of css classes, but is gobbledygook to a sight-impaired human reading the article.

It's probably something that can be fixed, but I'll open another issue for it.

**Comment by MasssiveJuice08** [comment]
We should implement a clear file-naming guideline for files and directories for the wiki, in order to improve the wiki's SEO rankings.

This is following my discussion on the Quartz discord which yielded some good advice (https://discord.com/channels/927628110009098281/1259427083021451264/1259427083021451264), and based off further research of my own.

I've opened the `SEO-Optimization` branch for implementing these proposed changes.

## Proposed Changes:
For all filenames, especially `.md`

- **all-lowercase**
- **no spaces**
   - **use hyphens** instead of spaces
   - **don't use underscores** instead of spaces - search engines don't read these
- **no special characters**, e.g., `?`, `%`,`#`, `/`, `|`, `:`, `;`, `•`

For images specifically:
- **use descriptive filenames**
   - e.g., use `my-new-black-kitten.jpg` instead of `IMG00023.JPG`

## Considerations:

- all articles would need to have `title` and `alias` frontmatter added to them in order to the titles in plain English. These would add capitalization and replace hyphens with spaces.

## References
- Google Developer Documentation Style Guide, _['Filenames and File Types'](https://developers.google.com/style/filenames)_
- Google Search Central, _['Google image SEO best practices'](https://developers.google.com/search/docs/appearance/google-images#descriptive-alt-text%20descriptive-titles-captions-filenames)_
- MDN Web Docs, _['Dealing with Files'](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files)_
- ThoughtCo., _['How to Name HTML Files'](https://www.thoughtco.com/naming-html-files-3466503)_
